MantidQt/CustomInterfaces/src/Tomography/TomographyROIViewQtWidget.cpp

```diff
@@ -709,33 +704,38 @@ void ImageROIViewQtWidget::updatePlay() {
   showProjectionImage(m_stackSamples, val, currentRotationAngle());
 }
 
-void ImageROIViewQtWidget::browseStackClicked() {
-  m_presenter->notify(IImageROIPresenter::BrowseStack);
+void TomographyROIViewQtWidget::browseStackClicked() {
+  m_presenter->notify(ITomographyROIPresenter::BrowseStack);
 }
 
-void ImageROIViewQtWidget::browseImageClicked() {
-  m_presenter->notify(IImageROIPresenter::BrowseImage);
+void TomographyROIViewQtWidget::browseImageClicked() {
+  m_presenter->notify(ITomographyROIPresenter::BrowseImage);
 }
 
-void ImageROIViewQtWidget::updateFromImagesSlider(int /* current */) {
-  m_presenter->notify(IImageROIPresenter::UpdateImgIndex);
+void TomographyROIViewQtWidget::updateFromImagesSlider(int /* current */) {
+  m_presenter->notify(ITomographyROIPresenter::UpdateImgIndex);
 }
 
-void ImageROIViewQtWidget::imageTypeUpdated(int /* idx */) {
-  m_presenter->notify(ImageROIPresenter::ChangeImageType);
+void TomographyROIViewQtWidget::imageTypeUpdated(int /* idx */) {
+  m_presenter->notify(TomographyROIPresenter::ChangeImageType);
 }
 
-void ImageROIViewQtWidget::rotationUpdated(int /* idx */) {
+void TomographyROIViewQtWidget::rotationUpdated(int /* idx */) {
   m_params.rotation =
       static_cast<float>(m_ui.comboBox_rotation->currentIndex()) * 90.0f;
-  m_presenter->notify(ImageROIPresenter::ChangeRotation);
+  m_presenter->notify(TomographyROIPresenter::ChangeRotation);
+}
+
+void TomographyROIViewQtWidget::autoCoRToolChanged(int /* idx */) {
+  // todo notify presenter that the tool is changed, however for now we only
+  // have one tool -> tomopy
```

Comment:
Yes this is connected to changing the tool method, and will be used when(if) there is more than one available.